beta/src/styles/sandpack.css

```
@@ -2,289 +2,267 @@
-  --theme-comment: #6a737d;
-  --theme-keyword: #d73a49;
-  --theme-tag: #22863a;
-  --theme-punctuation: #24292e;
-  --theme-definition: #6f42c1;
-  --theme-property: #005cc5;
-  --theme-static: #032f62;
-  --theme-string: #032f62;
+  --sp-colors-accent: #087ea4;
+  --sp-colors-clickable: #959da5;
+  --sp-colors-disabled: #24292e;
+  --sp-colors-error: #811e18;
+  --sp-colors-error-surface: #ffcdca;
+  --sp-colors-surface1: #fff;
+  --sp-colors-surface2: #e4e7eb;
+
+  --sp-syntax-color-plain: #24292e;
+  --sp-syntax-color-comment: #6a737d;
+  --sp-syntax-color-keyword: #d73a49;
+  --sp-syntax-color-tag: #22863a;
+  --sp-syntax-color-punctuation: #24292e;
+  --sp-syntax-color-definition: #6f42c1;
+  --sp-syntax-color-property: #005cc5;
+  --sp-syntax-color-static: #032f62;
+  --sp-syntax-color-string: #032f62;
 }
+
 html.dark .sp-wrapper {
-  --sp-colors-fg-active: #ffffff !important;
-  --sp-colors-fg-default: #999999 !important;
-  --sp-colors-fg-inactive: #343434 !important;
-  --sp-colors-bg-active: #343434 !important;
-  --sp-colors-bg-default: #16181d !important;
-  --sp-colors-bg-default-overlay: #16181d !important;
-  --sp-colors-bg-input: #242424 !important;
-  --sp-colors-accent: #6caedd !important;
-  --sp-colors-bg-error: #ffcdca !important;
-  --sp-colors-fg-error: #811e18 !important;
-  /* syntax */
-  --theme-plain: #ffffff;
-  --theme-comment: #757575;
-  --theme-keyword: #77b7d7;
-  --theme-tag: #dfab5c;
-  --theme-punctuation: #ffffff;
-  --theme-definition: #86d9ca;
-  --theme-property: #77b7d7;
-  --theme-static: #c64640;
-  --theme-string: #977cdc;
-}
-
-.sp-tabs,
-.sp-tab-button:hover {
-  background: none !important;
+  --sp-colors-accent: #149eca;
+  --sp-colors-clickable: #999;
+  --sp-colors-disabled: #fff;
+  --sp-colors-error: #811e18;
+  --sp-colors-error-surface: #ffcdca;
+  --sp-colors-surface1: #16181d;
+  --sp-colors-surface2: #343a46;
+
+  --sp-syntax-color-plain: #ffffff;
+  --sp-syntax-color-comment: #757575;
+  --sp-syntax-color-keyword: #77b7d7;
+  --sp-syntax-color-tag: #dfab5c;
+  --sp-syntax-color-punctuation: #ffffff;
+  --sp-syntax-color-definition: #86d9ca;
+  --sp-syntax-color-property: #77b7d7;
+  --sp-syntax-color-static: #c64640;
+  --sp-syntax-color-string: #977cdc;
+}
+
+/**
+ * Reset
+ */
+.sandpack--playground .sp-layout,
+.sandpack--playground .sp-tabs,
+.sandpack--playground .cm-activeLine,
+.sandpack .cm-editor,
+.sandpack--playground .cm-editor .cm-matchingBracket {
+  border: none;
+  background-color: transparent;
+}
+
+.sandpack--playground .cm-content {
+  caret-color: initial;
+}
+
+/**
+ * Layout
+ */
+.sandpack .sp-wrapper {
+  width: 100%;
+}
+.sandpack .sp-layout {
+  overflow: visible;
+  -webkit-mask-image: -webkit-radial-gradient(
+    var(--sp-colors-surface1),
+    var(--sp-colors-surface1)
+  ); /* safest way to make all corner rounded */
+
+  border-bottom-left-radius: 0.5rem;
+  border-bottom-right-radius: 0.5rem;
+}
+.sandpack .sp-layout > .sp-stack {
```

```suggestion
@media screen and (max-width: 768px)
.sp-layout>.sp-stack {
    height: auto;
    min-width: 100%!important;
}

.sandpack .sp-layout > .sp-stack {
```

May be this fixes the issue? i see this is missing with this branch?

---

thanks!